test.js

```diff
@@ -331,3 +331,15 @@ test('doesn\'t write to disk upon instanciation if and only if the store didn\'t
 	exists = fs.existsSync(conf.path);
 	t.is(exists, true);
 });
+
+test('`validate` option with invalid data', t => {
+	const conf = new Conf({cwd: tempy.directory(), validate: true});
+	fs.writeFileSync(conf.path, 'Non-JSON data');
+	t.deepEqual(conf.store, {});
```

Comment:
It will never throw because of this check in line 155:

```
if (error.name === 'SyntaxError') {
  return plainObject();
}
```

`JSON.parse` throws `SyntaxError` which is already caught. We can either try/catch the `JSON.parse` in line 143-146 and throw error there, which you have suggested not to do, or remove the check for `SyntaxError`.

Reply:
@sindresorhus what do you think about that?